### ISSUE TITLE
Job scheduled to run at 10 AM was scheduled and run at 1 AM

### ISSUE THREAD

**Issue by marco-c** [mozilla/telemetry-analysis-service#351]
I've scheduled my `top-signatures-correlations` job to run at 10 AM. Today it was scheduled and run at 1 AM:
```
Last scheduled date
2017-03-30 01:15
Last run date
2017-03-30 02:15
```

**Comment by marco-c** [comment]
Notice it was scheduled and run at 1 AM today (2017-03-30), but skipped 10 AM yesterday (2017-03-30).

**Comment by marco-c** [comment]
Today:
Start date - 2017-04-04 17:00
Last scheduled date - 2017-04-04 11:40
Last run date - n/a
Last terminated date - 2017-04-04 22:00

I've got an email at 22:00 UTC saying that the job at 11:40 UTC failed.

I'm not sure what happened to the 17 UTC job.

**Comment by jezdez** [comment]
@marco-c While investigating #326 I've found that ATMO's naive cron implementation mishandles the start date which leads to the start time of consecutive runs to slip slowly as it depends on previous runs instead on the originally saved start date. That's why your assumption of the job to always start at the start time can't currently be guaranteed. This is a bug and I'll fix it at the same time when I work on #372.

**Comment by jezdez** [comment]
This should be fixed since #379 was merged.
